### ISSUE TITLE
import error in main.py

### ISSUE THREAD

**Issue by whaler404** [mzweilin/EvadeML-Zoo#15]
Hello, I would like to ask if the datasets in the main program refer to the datasets library of Hugging Face. But the problem I'm having now is not being able to reference the module from datasets.

**Comment by mzweilin** [comment]
No, the `datasets` in the import path refers to the `datasets` folder: https://github.com/mzweilin/EvadeML-Zoo/tree/master/datasets
